packages/components/src/components/tabs/tabs.jsx

```diff
@@ -4,6 +4,7 @@ import { withRouter } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import Tab from './tab.jsx';
 import { useConstructor } from '../../hooks';
+import ThemedScrollbars from '../themed-scrollbars';
```

Comment:
We should import the JSX file rather than the _importer_  (= `themed-scrollbars/index.js`) we created so we don't bundle the CSS twice.

```suggestion
import ThemedScrollbars from '../themed-scrollbars/themed-scrollbars.jsx';
```